webapp-django/crashstats/crashstats/utils.py

```diff
@@ -342,17 +342,11 @@ def enhance_raw(raw_crash):
             raw_crash["AdapterDeviceName"] = result[1]
 
 
-#: Number of days to look at for versions in crash reports. This is set
-#: for two months. If we haven't gotten a crash report for some version in
-#: two months, then seems like that version isn't active.
-VERSIONS_WINDOW_DAYS = 60
-
-
 def get_versions_for_product(product="Firefox", use_cache=True):
     """Returns list of recent version strings for specified product
 
     This looks at the crash reports submitted for this product over
-    VERSIONS_WINDOW_DAYS days and returns the versinos of those crash reports.
```

Comment:
Fixed this typo. It was just sitting there. Its very existence was a blight upon this repository. Fixing it is probably the most important thing I did all year.